sebs/fission/config.py

```diff
@@ -0,0 +1,50 @@
+from sebs.faas.config import Config, Credentials, Resources
+from sebs.cache import Cache
+
+
+class FissionCredentials(Credentials):
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def initialize(config: dict, cache: Cache) -> Credentials:
```

Comment:
Check here how Fission deployment works for containers - maybe we need to pass Docker credentials?

Reply:
right now, I am more focused on code package rahtner than contaienr deployment.